src/main/ngapp/src/app/login/login.component.ts

```diff
@@ -53,4 +56,12 @@ export class LoginComponent implements OnInit {
        this.error = error.message;
      });
  }
+
+ sendPasswordResetEmail(email: string) {
+    if(!email) {
```

Comment:
should you be checking for empty string here?

Reply:
i was checking to see if the email field had been filled by user before sending email.
is there a better way/ place to do this?

Reply:
No the place is fine. I was just wondering if the condition should be " email === '' "?

Reply:
i think it should, actually. i'll make the change